README.md

```diff
@@ -1,2 +1,57 @@
-# My-Portfolio
-My personal portfolio
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> My Portfolio Project, Module 1: week 1: Day 3
+
+## Built With
+
+- Major languages : HTML & CSS
+- Frameworks : none
+
+## Getting Started
+
+**This project is create by @mrbhi. Go through, to see the content of the project Cheers!**
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Deployment
```

Comment:
- [ ] Please make sure that you add some necessary pieces of information for all your main header section or remove all empty and unnecessary header sections from your readme in order to make it more professional :+1: